.github/workflows/build.yml

```diff
@@ -295,6 +295,24 @@ jobs:
       working-directory: tests/FlatBuffers.Test.Swift
       run: sh SwiftTest.sh
 
+  build-swift-wasm:
+    name: Build Swift WASM
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Wasmer
+        uses: wasmerio/setup-wasmer@v1
+      - name: Install carton
+        run: |
+          git clone https://github.com/swiftwasm/carton
```

Comment:
Is there a specific need to build the main branch from source here? We can push breaking changes to the branch without warning. I recommend using [swiftwasm-action](https://github.com/swiftwasm/swiftwasm-action) instead, which has everything prebuilt and preinstalled, while also using only stable tagged versions.

Reply:
I actually couldn't make it work with [swiftwasm-action](https://github.com/swiftwasm/swiftwasm-action). I was doing something wrong for sure, i need to revisit this today

Reply:
If that doesn't work, you can also use `carton` Docker images hosted on GitHub: https://github.com/swiftwasm/carton/pkgs/container/carton

Reply:
@MaxDesiatov Thanks for the suggestion of using the container instead of the action! it should be working now

Reply:
No problem, I'm glad it works well for you